packages/dx-react-scheduler-material-ui/src/templates/views/horizontal/time-table/layout.jsx

```diff
@@ -1,51 +1,46 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import classNames from 'clsx';
-import TableMUI from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import { makeStyles } from '@material-ui/core/styles';
-import { cellsMeta, getWidthInPixels, getViewCellKey } from '../../../utils';
-import {
-  CELL_WIDTH, SMALL_CELL_WIDTH,
-  XS_CELL_WIDTH, SMALL_LAYOUT_MEDIA_QUERY, LAYOUT_MEDIA_QUERY,
-} from '../../../constants';
+import { Table } from '../../common/table';
+import { cellsMeta, getViewCellKey } from '../../../utils';
 
-const useStyles = makeStyles({
-  table: {
-    tableLayout: 'fixed',
-    minWidth: cellsNumber => getWidthInPixels(cellsNumber, CELL_WIDTH),
-    width: '100%',
-    [`${LAYOUT_MEDIA_QUERY}`]: {
-      minWidth: cellsNumber => getWidthInPixels(cellsNumber, SMALL_CELL_WIDTH),
-    },
-    [`${SMALL_LAYOUT_MEDIA_QUERY}`]: {
-      minWidth: cellsNumber => getWidthInPixels(cellsNumber, XS_CELL_WIDTH),
-    },
-  },
-});
+export class Layout extends React.PureComponent {
```

Comment:
Can we extract some base layout component to reuse it in both vertical and horizontal views? Now setCells routine, tests, Table rendering are duplicated. Only children will be different